README.md

```diff
@@ -1,6 +1,164 @@
 # NCIT Alumni Portal
```

Comment:
Instead lets go for Open Alumn Portal as project name. As we are looking forward to change repo name too once we get basic things done.

Reply:
sure